lib/anyway/rails/settings.rb

```diff
@@ -79,5 +79,7 @@ def app_root
     self.use_local_files ||= ::Rails.env.development?
     # Don't try read defaults when no key defined
     self.default_environmental_key = nil
+
+    self.suppress_required_validations = ENV["SECRET_KEY_BASE_DUMMY"]
```

Comment:
Anyway-specific env var must have precedence:

```suggestion
    self.suppress_required_validations = !!ENV["SECRET_KEY_BASE_DUMMY"] unless ENV.key?("ANYWAY_SUPPRESS_VALIDATIONS")
```